### PR TITLE
Improve rating algorithm and add more tests

### PR DESCRIPTION
**Pull request by ekumlin**
- Update the rating algorithm to better handle large claim counts
- Update a bunch of tests
- Fix some linting issues

### CODE REVIEW

**Comment by ekumlin** [comment]
@frosteeze Feel free to download this locally and see if the rating algorithm makes sense for various employers.

Good ones to look at are Amazon (huge claim count), GameStop (widely regarded as performing poorly but showing up as "Fair"), and Staples and United (small claim counts). But of course feel free to look at others as well...

I'd like to update the list in `EmployerRecord.test.ts` to make a really comprehensive set of possibilities so we can get a solid algorithm built around it. The one I have in this PR so far is a good start, but I'd like it to be refined as best we can.

File: src/common/EmployerRecord.ts [hunk 1]
```diff
@@ -8,61 +7,87 @@ export class EmployerRecord extends EmployerRecordBase {
...
+		let negatives: number = 0;
+		let positives: number = 0;
+		let score: number = 0;
+
+		if (citationRatings.length === 1) {
+			score = citationRatings[0];
+		} else if (citationRatings.length > 1) {
+			normalizedScore = 1;
+
+			let i: number = 0;
```
Comment (frosteeze) [inline src/common/EmployerRecord.ts:50]: Would it be possible to rename this variable to make it more readable?

Comment (ekumlin) [reply src/common/EmployerRecord.ts]: I've reformatted the algo completely so this is gone.

**Comment by ekumlin** [comment]
I've completed the new version of this algorithm.

If there are any issues with employers being shown with ratings that seem incorrect, please open a bug and reference the test case (in `EmployerRecord.test.ts`) which passes that you believe should fail, or whether there is a missing test case that should be added.

**Pull request merged**